### PR TITLE
[13.0][MIG] account_payment_term_partner_holiday: Migration to 13.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 13.0.

Please @joao-p-marques and @pedrobaeza can you review it?

@Tecnativa TT30526

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Review by dreispt — APPROVED** [review]

File: account_payment_term_partner_holiday/models/res_partner.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def is_date_in_holiday(self, date):
                     res_date_to = fields.Date.from_string(
                         "%s-%s-%s" % (date.year, res.month_to, day_to)
                     )
-                except:
+                except BaseException:
```
Comment (joao-p-marques) [inline account_payment_term_partner_holiday/models/res_partner.py]: As we are changing this, could this maybe be a more specific exception? I guess this should probably come from the conversion from `string` to `Date`, so you probably want to catch a `ValueError`: https://docs.python.org/3/library/datetime.html#datetime.datetime.strptime

**Review by joao-p-marques — CHANGES_REQUESTED** [review]
Code and functional review :+1: 

The only thing that is not so clear is that, at least in v13, the computed due date is not shown, so the user has no idea of what the result is (I don't remember if this was also the case for v12). It happens because Odoo hides that field if a payment term is set, so maybe it would make sense to force its visibility here? Something similar is done in https://github.com/OCA/account-invoicing/tree/13.0/account_invoice_date_due, that allows that field to be editable

**Comment by pedrobaeza** [comment]
No need to show the computed due date. That's out of the scope of the module.

**Pull request merged**